packages/web/src/services/AudiusBackend.js

```diff
@@ -2452,6 +2452,25 @@ class AudiusBackend {
     }
   }
 
+  static async recordIP() {
```

Comment:
let's make a new file `RecordIP.ts` in the `services/audius-backend` folder for this guy, so we're not adding more code to this poor untyped file